src/Genesis.sol

```diff
@@ -56,13 +62,15 @@ contract Genesis is ERC721, Ownable {
         totalSupply = currSupply;
     }
 
-    function checkForMergeAndUpdate() public {
-        if (genesisMergeBlock == 0 && mergeHasOccured()) {
+    function assertPoS() public {
```

Comment:
i was thinking we could make this a function anyone could call just so we can keep updating the blocks (so we can guarantee not missing the POS block)